api/backup_storage.go

```diff
@@ -276,7 +276,7 @@ func (e *EverestServer) UpdateBackupStorage(ctx echo.Context, backupStorageName
 		})
 	}
 
-	params, err := validateUpdateBackupStorageRequest(ctx, bs, secret, namespaces, e.l)
+	params, err := e.validateUpdateBackupStorageRequest(ctx, backupStorageName, bs, secret, namespaces, e.l)
```

Comment:
Just a thought, leaving up to you.  

We could have a universal method to extract the name and the namespace from the resources metadata to avoid passing additional parameters like this. For example, the `nameFromDatabaseCluster` method does a very similar thing, we could change the error strings to be not the databasecluster's specific and reuse it.

Reply:
yes, that's a good idea. I may look into it in a separate PR.